tuist/tuist
# Embed static frameworks with bundled resources instead of generating a new bundle

Relates to <https://github.com/tuist/tuist/issues/5761>

### Short description 📝

PR introduces support of static frameworks with bundled resources on `tuist` side. Xcode 15 [supports](https://developer.apple.com/documentation/xcode/creating-a-static-framework) it.

### How to test the changes locally 🧐

- `tuist` won't generate a bundle target if a static framework contains resources
- Resources appear in the **Copy Bundle Resource** build phase
- `Bundle.module` is generated as shown [here](https://forums.developer.apple.com/forums/thread/751477?answerId=811319022#811319022).
- Static frameworks with resources are embedded to the app target like dynamic ones

### Contributor checklist ✅


### Reviewer checklist ✅
